twittermap/public/javascripts/map/controllers.js

```diff
@@ -671,7 +671,7 @@ angular.module('cloudberry.map', ['leaflet-directive', 'cloudberry.common'])
           if($scope.doSentiment) {
             $scope.infoPromp = "Score";  // change the info promp
```

Comment:
hmm, I think we still need to. `sentiment` is not always enabled. If user toggle off the `sentiment`, we will still need to reset it. One `config.mapLegend` is not enough.